src/encoded/types/dataset.py

```diff
@@ -1383,6 +1383,25 @@ class FunctionalCharacterizationSeries(Series):
     def elements_references(self, request, related_datasets):
         return request.select_distinct_values('elements_references', *related_datasets)
 
+    @calculated_property(schema={
```

Comment:
Should these new calculated properties also be added to the facets and columns for Functional Characterization Series?

Reply:
Idan has now added to the facet.  Adding to columns is bringing up issues with the columns that are larger than this ticket

Reply:
Sounds good, I see assay_title added. assay_type is not needed as a facet correct? Only needs to exist as a property